mkn.sh

```
@@ -0,0 +1,27 @@
+shopt -s expand_aliases
+alias cls="clear; printf '\033[3J'"
+cls
+
+set -e
+
+TEST="-M tests/core/numerics/ion_updater/test_multi_updater.cpp"
+XFILE="${XFILE:-res/mkn/clang_cuda}"
+[ -d /opt/rocm/bin ] && XFILE="res/mkn/hip"
+
+ARGS="${TEST} -P mkn.base=gpu_"
+[ -n "XFILE" ] && ARGS+=" -x ${XFILE}"
```

Shellcheck

[error] Argument to -n is always true due to literal strings.

(SC2157)